razorpay/utility/utility.py

```diff
@@ -35,5 +35,28 @@ def verify_signature(self, body, signature, key):
 
         generated_signature = dig.hexdigest()
 
-        if not hmac.compare_digest(generated_signature, signature):
-            raise SignatureVerificationError('Payment Signature Verification Failed')
+        if sys.version_info[0:3] < (2, 7, 0):
+            result = self.compare_string(generated_signature, signature)
+        else:
+            result = hmac.compare_digest(generated_signature, signature)
+
+        if not result:
+            raise SignatureVerificationError(
+                'Payment Signature Verification Failed')
```

Comment:
Change message to s/Payment/Razorpay since we are using this for webhooks as well.